experimental/dds/ot/ot/README.md

```diff
@@ -32,7 +32,7 @@ npm i @fluid-experimental/ot
 
 ## API Documentation
 
-API documentation for **@fluid-experimental/ot** is available at <https://fluidframework.com/docs/api/v1/ot>.
+API documentation for **@fluid-experimental/ot** is available at <https://fluidframework.com/docs/apis/ot>.
```

Comment:
[Build 227406](https://dev.azure.com/fluidframework/public/_build/results?buildId=227406&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=3f6c1bb4-e2b1-5180-b7f3-9a06f477b8ee&l=14)

@Josmithr 

Pipeline is complaining about extraneous modified files `api-docs-link-section.md` and `library-package-readme.md`, but I think I should also revert the changes (`api/v1`  back to `/apis`) in other `README` files?

Reply:
@chentong7 is currently working on resolving these. I would recommend just reverting these changes on your PR for now.